### ISSUE TITLE
Import Wellcome Trust grants data into Donations List Website (XLSX available)

### ISSUE THREAD

**Issue by vipulnaik** [vipulnaik/donations#8]
See https://wellcome.ac.uk/funding/managing-grant/grants-awarded

XLSX file at https://wellcome.ac.uk/sites/default/files/wellcome-grants-awarded-2000-2016.xlsx

**Comment by riceissa** [comment]
I have started this at https://github.com/riceissa/wellcome-trust

**Comment by riceissa** [comment]
I think this is done (with the usual FIXMEs needing to be filled). Some notes I made in chat (this was over a month ago so I don't have a good memory):

- wellcome trust amounts are in pounds
- also not sure about cause area here
- there is a "grant type" column with the following values https://gist.github.com/riceissa/0b4108e3599d714dd9459725831566cd
- there is also a "panel" column with the following values https://gist.github.com/riceissa/95f8e23680c2da4de7cb723b42f703c6
so if you want to somehow use those to map to cause areas, that is one idea

**Comment by vipulnaik** [comment]
Can we use fixer.io to do currency conversion for this one, like you did for #6?

**Comment by riceissa** [comment]
Yes, I'll use Fixer.io. I think you meant #25 (Unbound Philanthropy) rather than #6 (Ford).

**Comment by riceissa** [comment]
Okay, currency conversion is done.

**Comment by vipulnaik** [comment]
Inserted, next step is cleanup.
